src/open_inwoner/cms/cases/views/status.py

```diff
@@ -383,6 +386,10 @@ def get_upload_info_context(self, case: Zaak):
                     zt_statustype_config.document_upload_description
                 )
 
+        # disable document upload message
+        if self.has_recently_uploaded_documents:
```

Comment:
This isn't how this should be solved. The upload-message should only be hidden directly after you've uploaded your documents, and not be related to if you have recent documents or not.

Reason is that the municipality can also upload documents, so ensure we only hide the upload description after returning from CaseDocumentUploadFormView. I'd personally do this based on a GET-argument after a succesful upload

Reply:
Makes sense, I didn't consider this case.